### PR TITLE
fix: DIA-1489: Multiskill prompt autorefinement fixes

### PR DESCRIPTION
**Pull request by niklub**

### CODE REVIEW

File: src/label_studio_sdk/label_interface/control_tags.py [hunk 1]
```diff
@@ -462,6 +462,23 @@ def label(
             )
         else:
             return self._label_simple(to_name=to_name, *args, **kwargs)
+        
+    def get_labels(self, regions: List[Dict]):
+        """
+        Returns the simplified representation of the label. Sort of a reverse to label() method to retrieve an input `label` from an output regions
```
Comment (matt-bernstein) [inline src/label_studio_sdk/label_interface/control_tags.py:468]: can you put an example in this docstring? It's hard to picture what the output looks like

**Review by matt-bernstein — APPROVED** [review]

File: src/label_studio_sdk/label_interface/control_tags.py (hunk 1 above)
Comment (matt-bernstein) [reply src/label_studio_sdk/label_interface/control_tags.py:468]: or add a test

Comment (niklub) [reply src/label_studio_sdk/label_interface/control_tags.py:468]: sure will do both

Comment (niklub) [reply src/label_studio_sdk/label_interface/control_tags.py:468]: done

**Pull request merged**